sdk/managementgroups/arm-managementgroups/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 2.0.2 (Unreleased)
+## 2.0.2 (2022-12-02)
 
```

Comment:
We have issues for this changelog and we need to fix
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2030888&view=logs&j=fad5caa0-24eb-51d7-9ee8-d55d6e9ffe5a&t=4b903053-dc5b-56d4-4838-edf64acaab7d&l=28